requirements.txt

```diff
@@ -0,0 +1 @@
+spade
```

Comment:
Wouldn't specifying a version be safer? It could save us from e.g. unknowingly upgrading to a newer version which could be incompatible with the one we started with.